SCALERS.md

```diff
@@ -0,0 +1,57 @@
+# Scaler Governance
+
+This document describes how KEDA handles governance of built-in scalers and how it relates to external scalers maintained by the community.
+
+## Scalers in KEDA
+
+Each event source has unique characteristics on how KEDA needs to connect to it and the types of metrics available to drive scaling. KEDA is made up of many *scalers*, or simple extensions to the core KEDA operator that allows it to connect to an event source.
+
+> 📝 You can find a full list of supported scalers in [our documentation](https://keda.sh/docs/latest/scalers/).
+
+Scalers are shipped with KEDA in two ways:
+
+- **Built-in Scalers** - Scalers are packaged and shipped as part of the core KEDA runtime
+- **External Scalers** - Scalers that are running outside of KEDA but are capable of being used to scale workloads with. Workloads can be scaled by using the "External" or "External Push" scalers which integrate with the external scalers by using gRPC to drive scale.
+
+### Choosing the right model for a scaler
+
+When introducing a new scaler, it is important to choose the right model for it - Should it be a built-in scaler or an external scaler?
+
+The general rules of thumb for built-in scalers are the following:
+
+- They must live in the [core KEDA repository](http://github.com/kedacore/keda)
+- It follows the [KEDA core release cycle](https://github.com/kedacore/keda/blob/main/ROADMAP.md#upcoming-release-cycles)
+- It has no additional infrastructure requirements
+- It is general-purpose and not specific to a single end-user
+- It is written in Go
+
+If all of the above are true, then then it is a potential case for a new built-in scaler. We recommend opening a feature request to propose and discuss it before doing the actual implementation though. This is to avoid throwing things away if it does not meet the needs after all.
+
+### Requirements for a built-in scaler
+
+In order to contribute a scaler to KEDA's core, it must meet the following requirements:
+
+- A scaler MUST be written in Go
+- A scaler owner MUST be assigned (individual(s)/company) and will be tagged for bug reports
+- A scaler MUST have e2e and unit tests to guarantee its quality
+- A scaler MUST be backwards compatible until the next KEDA major version
+- A scaler CAN be deprecated when:
+  - Maintainers believe it is best for the project, after a vote
+  - The scaler owner(s) are not responsive and support volume is too high
+  - The technology is proven to be no longer used/abandoned
+- Scaler MUST follow KEDA release cycles
+- A scaler MUST be supported
+
+### Choosing a home for external scalers
```

Comment:
Something we can evaluate would be to introduce a `keda-contrib` org for external scalers to centralize them; but give external maintainers the required permissions.

Although I don't think we need that at this point in time.

Reply:
I like the idea of having another org more open but maybe we could just invite them to current org and create different teams inside.
Wdyt?

Reply:
I mean, using current org we "create a group feeling", people could be more engaged being part of kedacore.

Reply:
> I like the idea of having another org more open but maybe we could just invite them to current org and create different teams inside.

If we do that, then there is no longer a distinction between "official" external scalers and the ones maintained by the community as pointed out in the doc.

I'll resolve this and we can still introduce this later on unless @zroubalik has some thoughts on this.